src/plugins/data_source/opensearch_dashboards.json

```diff
@@ -6,5 +6,5 @@
   "ui": true,
   "requiredPlugins": ["opensearchDashboardsUtils"],
   "optionalPlugins": [],
-  "extraPublicDirs": ["common","common/data_sources", "common/util"]
+  "extraPublicDirs": ["common","common/data_sources", "common/data_connections", "common/util"]
```

Comment:
is data_connections always available?

then i think this will cause a dependency tree that data plugin is dependent on the connections work

Reply:
> is data_connections always available

this depends on whether the plugin that depends on data_source plugin calls it required or optional

> this will cause a dependency tree that data plugin is dependent on the connections work

this is not used by the data plugin, i exported it for cloudwatch implementation. it should be exported because `common/data_connections` should behave similar like `common/data_sources`, they are same level concepts